addons/sale_stock/tests/test_create_perf.py

```diff
@@ -38,7 +38,7 @@ def setUpClass(cls):
     @users('admin')
     @warmup
     def test_empty_sale_order_creation_perf(self):
-        with self.assertQueryCount(admin=34):
+        with self.assertQueryCount(admin=36):
```

Comment:
2 queries added for getting the analytic account `search` in `Model<account.analytic.default>.account_get` + another one?